app/api/routes/summaries.py

```
@@ -0,0 +1,186 @@
+    """
+    task_result = AsyncResult(job_id)
+
+    if task_result.state in ["SUCCESS", "FAILURE"]:
+        raise HTTPException(
+            status_code=400, detail="이미 완료되거나 실패한 작업은 취소할 수 없습니다"
+        )
+
+    try:
+        task_result.revoke(terminate=True)
+        logger.info(f"[API] Celery task cancelled: {job_id}")
+        return {"message": "작업이 취소되었습니다", "job_id": job_id}
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)